test/inductor/test_torchinductor.py

```diff
@@ -3903,6 +3916,7 @@ def fn(x, y):
         if self.device != "cpu":
             assertGeneratedKernelCountEqual(self, 1)
 
+    @config.patch(fx_graph_cache=False)
```

Comment:
Would it be easier to increment this counter as part of fx graph caching loading ?

Reply:
@eellison , you mean save away the delta for that counter with the cache entry so that I can bump it by the same amount on a cache hit? Sure, I could do that. Do you mean it would be nice to hit the cache code path for these particular tests? Or this particular counter is useful for other purposes? Or maybe all these exclusions are just an eyesore.

Reply:
It's not that they're an eyesore but I think the more caching works out of the box with existing tests the better